src/lib/components/product.svelte

```diff
@@ -5,43 +5,80 @@
   export let image: string
   export let name: string
   export let brand: string
+  export let sizeAndDegrees: string | undefined
+  export let region: string | undefined
 </script>
 
-<!-- <a href="#"> -->
-<div in:fade={{ duration: 300, delay: index * 150 }} class="product">
-  <img src={image} alt="" />
-  <h2>{brand}</h2>
-  <h4>{name}</h4>
+<div in:fade={{ duration: 300, delay: index * 150 }} class="card">
+  <img src={image} alt={name} />
+  <div class="content">
+    <h2>{brand} {name}</h2>
```

Comment:
This is 3rd time I comment on this - you are not allowed to add spaces like this, it is unsafe

Reply:
this is corrected in the other pr especially because there is no brand anymore